src/forms/FormFieldAddress/AddressAutocomplete/AddressAutocomplete.styled.tsx

```diff
@@ -0,0 +1,48 @@
+import styled from "styled-components";
+import theme from "@root/theme";
+import { StyledTextField } from "@root/forms/FormFieldText/FormFieldText.styled";
+
+// Components
+import ClearIcon from "@mui/icons-material/Clear";
+
+export const LocationSearchInputWrapper = styled.div`
+	&.mapCoordinates {
+		margin-bottom: 8px;
+		margin-top: 15px;
+	}
+`;
+
+export const StyledInputSearch = styled(StyledTextField)`
+  min-width: 684px;
+
+  &.MuiTextField-root {
+    display: flex;
+  }
+`;
+
+export const StyledClearIcon = styled(ClearIcon)`
+  cursor: pointer;
+  color: ${theme.newColors.almostBlack["100"]};
+`;
+
+export const SuggestionsContainer = styled.div`
+  border: ${theme.borders.fieldGray};
+  font-family: ${theme.fontFamily};
+`;
+
+export const SuggestionsDescriptionContainer = styled.div`
+  background-color: ${({ isSuggestionActive }) => isSuggestionActive ? theme.colors.gray200 : "white"};
+  cursor: pointer;
+	width: ${({ fieldSize }) => fieldSize};
```

Comment:
We need to remove 2px from the fieldSize because of the 1px border, else the container gets on top of the border, this can be solved if we move some of the logic between components, for example: Here we can keep width as 100%, and move the padding from SuggestionDescription up to this component. That way the one applying the padding is the parent and not the child. Also this has some extra indentation